src/Phinx/Config/Config.php

```diff
@@ -538,4 +538,12 @@ public function offsetUnset($id): void
     {
         unset($this->values[$id]);
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function getSeederTemplateFile(): ?string
+    {
+        return $this->values['templates']['seedTemplateFile'] ?? null;
```

Comment:
This config name should probably match the getter function name.

Reply:
> This config name should probably match the getter function name.

@othercorey Good that You raised this question - I was not sure, is there any kind of naming conventions in the project, how to name this in the config and how to name the getter...

Do You have any suggestions on the path of this option in the config or it's fine and I should just rename the getter? "getSeedTemplateFile()" is ok, or should I add "templates" to the getter name somehow?

Reply:
I suggest making the config `['templates']['seed']` or `['templates']['seedFile']` and the getter `getSeedTemplate()` or `getSeedTemplateFile()`.

Depends on whether other templates have the `File` suffix.

Reply:
> I suggest making the config `['templates']['seed']` or `['templates']['seedFile']` and the getter `getSeedTemplate()` or `getSeedTemplateFile()`.
> 
> Depends on whether other templates have the `File` suffix.

@othercorey Ok, I changed to `['templates']['seedFile']` and `getSeedTemplateFile()`